features/step_definitions/user_steps.rb

```diff
@@ -88,6 +88,20 @@
   step %{confirmation emails have been delivered}
 end
 
+Given(/^the user "([^"]*)" disallows co-creators$/) do |login|
```

Comment:
Should these be in https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/preferences_steps.rb with the other steps for setting preferences instead?